aptos-move/e2e-move-tests/src/tests/state_metadata.rs

```diff
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use crate::MoveHarness;
+use aptos_sdk::types::get_apt_primary_store_address;
```

Comment:
Sigh... all our stuff is so randomly interconnected.  We should someday get rid of all these single function carryovers.

Reply:
maybe I should move it somewhere... you are right